idunn/places/poi.py

```diff
@@ -80,6 +77,19 @@ def get_contribute_url(self):
         except ValueError:
             return None
 
+    def get_source(self):
+        return PoiSource.OSM
+
+
+class OsmPOI(POI):
+    def get_source(self):
+        return PoiSource.OSM
```

Comment:
This seems like an equivalent to the base class `Poi`. Maybe that's this base class that should be renamed into `OsmPOI`? Otherwise we could make `Osm` into an abstract class to explicitly deprecate direct use and avoid confusion.

Reply:
Ok we can use OSM as an abstract class.  there are three type : OSMPoi, TripadvisorPoi and BragiPoi. The weird thing is that BragiPoi can be a OSMPoi or a TripadvisorPoi. I am not really happy with what i did because everything is mix up